go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -476,7 +497,7 @@ func (n *NodeController) handleHybridOverlayMACIPChange(node *kapi.Node) error {
 
 // EnsureHybridOverlayBridge sets up the hybrid overlay bridge
 func (n *NodeController) EnsureHybridOverlayBridge(node *kapi.Node) error {
-	if atomic.LoadUint32(&n.initialized) == 1 {
+	if atomic.LoadUint32(&n.initialized) >= 1 {
```

Comment:
can you change the atomic comparisons to constants? Now you are using a new value of 2, but I'm not sure what that indicates (without referring to your comment in the struct). So maybe const iota with their names suggesting what the event is you are looking for would be clearer?